### DISCUSSION TITLE
봇 안켜짐

### DISCUSSION THREAD

**Discussion by catpooll** [Lastorder-DC/chatreader-kor#50] [문의] [answered]
오늘 채팅읽어주는봇이 작동이안되는데
봇을 안켰거나 점검이거나 수정인건가요

**Comment by Lastorder-DC** [comment, marked as answer]
19b4c4f943e515971e6a615dce5d0e765edf99ce 사용 불가능한 TAPIC 대신 tmi.js로 교체되었습니다.
